pkg/consts/consts.go

```diff
@@ -41,5 +41,7 @@ var (
 	NewBaseHashFunc = sha256.New
 
 	// DataCommitmentBlocksLimit is the limit to the number of blocks we can generate a data commitment for.
+	// NOTE: this is no longer used as we're moving towards Blobstream X. However, we're leaving it
+	// here for backwards compatibility purpose until it's removed in the next breaking release.
```

Comment:
Question: I'm curious to know the impact of making this constant public versus keeping it private

Reply:
you mean exported vs unexported?

Reply:
> you mean exported vs unexported?

Yes

Reply:
it was public because we're using it in the Blobstream module, however, we're deprecating that so we can remove it or do anything with it really